frontend_build/src/webpack.config.prod.js

```diff
@@ -21,7 +21,9 @@ for (var i=0; i < bundles.length; i++) {
       compress: {
         warnings: false
       }
-    })
+    }),
+    // optimize module ids by occurence count
+    new webpack.optimize.OccurrenceOrderPlugin()
```

Comment:
I got rid of this on purpose:

https://webpack.js.org/guides/migrating/#occurrenceorderplugin-is-now-on-by-default